tests/query/parser/test_query.py

```diff
@@ -64,13 +63,30 @@ def with_required(condition: Optional[Expression] = None) -> Expression:
     return required
 
 
+DEFAULT_TEST_QUERY_CONDITIONS = [
+    "timestamp >= toDateTime('2021-01-01T00:00:00')",
+    "timestamp < toDateTime('2021-01-02T00:00:00')",
+    "project_id = 1",
+]
+
+# If these are the only WHERE conditions we can do the simplest thing
+PRECOMPUTED_TEST_QUERY_CONDITIONS_STR = " AND ".join(DEFAULT_TEST_QUERY_CONDITIONS)
+
+
+def prepend_condition_str_to_default(condition: str) -> str:
```

Comment:
This is a style nit, but I would change this function to take an arbitrary number of conditions, and then for the base case you could call it with no arguments.

```
def prepend_condition_str_to_default(*conditions: str) -> str:
    return " AND ".join(list(conditions) + DEFAULT_TEST_QUERY_CONDITIONS)
```

Then you don't need `PRECOMPUTED_TEST_QUERY_CONDITIONS_STR` and every test can use `prepend_condition_str_to_default`

You don't have to change this if you don't want, it feels like it simplifies things a bit.

Reply:
made this change, also changed the name of the method to `snql_conditions_with_default`